src/test/java/gov/usgs/earthquake/natsws/WebSocketNotificationReceivedTest.java

```diff
@@ -65,7 +65,7 @@ public void receiveNotification(Notification notification) {
       throw e;
     }
 
-    Thread.sleep(60000);
+    Thread.sleep(31000);
```

Comment:
Is this just waiting for websocket connection?  What if you use a listener sync object and wait for connection onOpen?

Reply:
this is vestigial; the sleep was temporary to verify reconnect behavior. the sleep is now removed.